features/DocumentController.feature

```diff
@@ -194,6 +194,21 @@ Feature: Document Controller
 
   # document:exists ============================================================
 
+  @mappings
+  Scenario: Check document existence
```

Comment:
A better test scenario would be one where at least 4 documents are checked in the same time with 2 existing and 2 not existing so you are testing both cases with more than 1 documents (which is the purpose of mExists, test documents by batch)